### PR TITLE
Auto run playbook doc

### PR DESCRIPTION
**Pull request by antazoey**

### CODE REVIEW

File: playbooks/code42_alert_response_playbook.json [hunk 1]
```diff
@@ -7121,10 +7120,10 @@
         "schema": 4,
         "version": "4.10.3.51237"
     },
-    "create_time": "2021-06-21T18:30:08.156052+00:00",
+    "create_time": "2021-06-21T18:31:15.108522+00:00",
     "draft_mode": false,
     "labels": [
-        "*"
+        "code42 alerts"
```
Comment (antazoey) [inline playbooks/code42_alert_response_playbook.json:7126]: This is the change that prompted the regen of this playbook

File: playbooks/code42_alert_response_playbook.json [hunk 2]
```diff
@@ -4348,7 +4347,7 @@
                                     {
                                         "column_name": "Risk Tag",
                                         "column_order": 9,
-                                        "data_path": "action_result.data.*.riskFactors",
+                                        "data_path": "action_result.data.*.riskFactors.*.tag",
```
Comment (antazoey) [inline playbooks/code42_alert_response_playbook.json:4350]: this is an automatic change from a fix that was made to the connector

File: playbooks/code42_alert_response_playbook.json [hunk 3]
```diff
@@ -7018,7 +7017,7 @@
                         "callback_code": "# read-only block view not available",
                         "callback_start": 1,
                         "callsback": false,
-                        "connected_to_start": false,
+                        "connected_to_start": true,
```
Comment (antazoey) [inline playbooks/code42_alert_response_playbook.json:7020]: ??? IDK

File: README.md [hunk 4]
```diff
@@ -28,3 +28,27 @@ The 'on poll' functionality first ingests the past 30 days of Code42 alerts (or
...
+
+## Playbook
+
+### Installation
+
+The playbook lives in the directory `playbooks/` and is an example of how to respond to a Code42 Alert.
+
+Install the playbook by first creating a tar of the `.py` and `.json` files.
+
+```bash
```
Comment (peterbriggs42) [inline README.md:40]: I actually wrote `make playbook` command for this...meant to update the docs with instructions, my bad.

File: playbooks/code42_alert_response_playbook.json (hunk 3 above)
Comment (peterbriggs42) [reply playbooks/code42_alert_response_playbook.json:7020]: Was this also an automatic change?

**Review by peterbriggs42 — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (antazoey) [reply README.md:40]: Yes I know, I could call that out, but I was trying to make it more generic in the readme for customers so they know what is going on and are probably quite familiar tarring things up, if they have to do this a lot.

I can call that out though.

**Comment by antazoey** [comment]
@peterbriggs42 Yes, that change was either automatic or is related to change of switching the event label... I am not sure! I can't find any docs on it.

When making the playbook Active, no changes to the JSON file occurred, so I don't think it's that.

**Pull request merged**